### ISSUE TITLE
DP and AD tags in VCF

### ISSUE THREAD

**Issue by thedam** [hellbelly/BS-Snper#16]
Hey,
in .vcf there are many tags like this:

chr1	924310	.	C	G	49	PASS	**DP=5**;ADF=0,5;ADR=0,12;**AD=0,17**;	GT:**DP**:ADF:ADR:**AD**:BSD:BSQ:ALFR	1/1:**5**:0,5:0,12:**0,17**:0,0,0,5,12,0,0,0:0,0,0,39,39,0,0,0:0.000,3.400
chr1	924321	.	C	G	58	PASS	**DP=8**;ADF=0,8;ADR=0,13;**AD=0,21**;	GT:**DP**:ADF:ADR:**AD**:BSD:BSQ:ALFR	1/1:**8**:0,8:0,13:**0,21**:0,0,0,8,13,0,0,0:0,0,0,38,39,0,0,0:0.000,2.625
chr10	71390271	.	C	T	1000	PASS	DP=33;ADF=0,0;ADR=6,27;AD=6,27;	GT:DP:ADF:ADR:AD:BSD:BSQ:ALFR	0/1:33:0,0:6,27:6,27:0,0,32,0,0,27,6,0:0,0,38,0,0,39,39,0:0.182,0.818

so in short:
```
DP=5, AD=0,17   - incorrect
DP=8, AD=0,21   - incorrect
DP=33, AD=6,27 - correct
```
DP should be sum of ADs, yes? Why in some cases the values are wrong?

**Comment by thedam** [comment]
also such example:
`chr1    779047  .       G       A       1000    PASS    DP=15;ADF=0,15;ADR=0,0;AD=0,15  GT:DP:ADF:ADR:AD:BSD:BSQ:ALFR   0/1:15:0,15:0,0:0,15:15,0,0,0,0,0,0,42:39,0,0,0,0,0,0,39:0,1
`
Why `GT=0/1`, and `AD=0,15`? Shouldn't it be `1/1`?

**Comment by bsiranosian** [comment]
Can confirm I'm seeing a lot of heterozygous calls that should be homozygous variant. If all the information is coming from the reverse strand, BS-Snper only calls homozygous variant if there are few to none reads coming from the forward strand, just by chance. I haven't looked at the source yet but this is probably a fixable bug.
